web/src/browser_history.js

```diff
@@ -13,7 +14,7 @@ export const state = {
     // hashes are changed without calling `hashchanged` in many ways.
     spectator_old_hash: hash_util.is_spectator_compatible(window.location.hash)
         ? window.location.hash
-        : "#",
+        : `#${user_settings.default_view}`,
```

Comment:
This change generates uglier URLs, and I think is unnecessary given the later commit to turn off the browser behavior, so I think we can drop this commit?

Reply:
Yes.

Reply:
Oh! I recall now that it was the prevent all messages from scrolling to the top.

Reply:
#25759